compatibility/history-proxy/src/index.js

```diff
@@ -0,0 +1,63 @@
+const { Logger } = require('@dojot/microservice-sdk');
```

Comment:
Add process event listener to _unhandledRejection_ and _uncaughtException_ to finish services when a unexpected behavior occurs 

https://github.com/dojot/dojot/blob/master/storage/time-series/influxdb/retriever/index.js#L42